### ISSUE TITLE
[Accessibility] context menus do not respond to applications key/shift-f10

### ISSUE THREAD

**Issue by ethindp** [godotengine/godot#117593]
### Tested versions

4.6.1

### System information

Godot v4.6.1.stable - Windows 10 (build 19044) - Multi-window, 1 monitor - Direct3D 12 (Forward+) - dedicated Intel(R) Iris(R) Xe Graphics (Intel Corporation; 27.20.100.9664) - 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz (8 threads) - 39.84 GiB memory

### Issue description

In docs such as the scene editor or file system doc, it is impossible to press applications or shift-f10 to open the context menu for a selected item. This is a miner annoyance, except when screen readers decide to be annoying and refuse to focus the mouse on what it is the keyboard is focused on (although this is more an SR bug).

### Steps to reproduce

1. Open a Godot project
2. Go to the scene or FS docs
3. Try to access the context menu when focused on something using the applications key (if you have one) or shift-f10 (if you lack an applications key like myself). Nothing will happen that is detectable by a screen reader.

### Minimal reproduction project (MRP)

N/A

**Comment by ethindp** [comment]
Ah okay, would it be worth fixing this for all menus?
